package.json

```diff
@@ -6,7 +6,7 @@
   "main": "lib/index.js",
   "types": "lib/index.d.ts",
   "scripts": {
-    "test": "TS_NODE_PROJECT='./src/test/tsconfig.commonjs.json' TS_NODE_TRANSPILE_ONLY=true ./node_modules/.bin/mocha --require ts-node/register --require mock-local-storage --require jsdom-global/register --allow-uncaught --colors --reporter mocha-reporter --timeout 5000 src/test/*.ts",
+    "test": "TS_NODE_PROJECT='./src/test/tsconfig.commonjs.json' TS_NODE_TRANSPILE_ONLY=true ./node_modules/.bin/mocha --require ts-node/register --require mock-local-storage --require jsdom-global/register --allow-uncaught --colors --reporter mocha-reporter --timeout 5000 src/test/*.ts src/test/crux-gateway/*.ts src/test/crux-messenger/*.ts",
```

Comment:
there is no such directory right? `test/crux-gateway`

Reply:
ok seems like your test case is in that directory. move it to crux-messenger

Reply:
There is `src/test/crux-gateway/asymmetric-encryption.ts`

Reply:
move it to test/crux-messenger

Reply:
@SanchayPahalwani you haven't addressed this

Reply:
Done